exoscale/resource_exoscale_compute.go

```diff
@@ -551,6 +570,11 @@ func resourceComputeUpdate(d *schema.ResourceData, meta interface{}) error {
 		req.DisplayName = d.Get("display_name").(string)
 	}
 
+	if d.HasChange("hostname") {
```

Comment:
Should you add a `d.SetPartial("hostname")` like for `display_name`?

Reply:
Good catch, missed that one. Fixed, thanks! 😉